js/renovation/button.tsx

```diff
@@ -19,10 +16,10 @@ const getImageContainerJSX = (source: string) => {
 const stylingModes = ['outlined', 'text', 'contained'];
 const defaultClassNames = ['dx-button'];
 
-const getCssClasses = (model: any) => {
+const getCssClasses = (model: ButtonInput) => {
     const { text, icon, stylingMode, type } = model;
-    const classNames = defaultClassNames.concat(model.classNames);
-    const isValidStylingMode = stylingModes.indexOf(stylingMode) !== -1;
+    const classNames = defaultClassNames.concat(model.classNames || []);
+    const isValidStylingMode = stylingMode && stylingModes.indexOf(stylingMode) !== -1;
```

Comment:
`stylingModes?.indexOf(stylingMode)`

Reply:
We will get `Argument of type '"outlined" | "text" | "contained" | undefined' is not assignable to parameter of type 'string'.
  Type 'undefined' is not assignable to type 'string'.ts(2345)` in this case